hooks/conf_regen/06-slapd

```diff
@@ -101,13 +102,14 @@ do_pre_regen() {
     rm -f "$tmp_backup_dir_file"
 
     # Define if we need to migrate from hdb to mdb
-    curr_backend=$(grep '^database' /etc/ldap/slapd.conf 2> /dev/null | awk '{print $2}')
-    if [ -e /etc/ldap/slapd.conf ] && [ -n "$curr_backend" ] \
-        && [ $curr_backend != 'mdb' ]; then
-        backup_dir="/var/backups/dc=yunohost,dc=org-${curr_backend}-$(date +%s)"
-        mkdir -p "$backup_dir"
-        slapcat -b dc=yunohost,dc=org -l "${backup_dir}/dc=yunohost-dc=org.ldif"
-        echo "$backup_dir" > "$tmp_backup_dir_file"
+    if [ -e /etc/ldap/slapd.conf ]; then
+        curr_backend=$(grep '^database' /etc/ldap/slapd.conf | awk '{print $2}')
+        if [ "$curr_backend" != 'mdb' ]; then
```

Comment:
What if $curr_backend is empty string `""` ? Are you sure it secure to remove `[ -n "$curr_backend" ]`
```suggestion
        if [ "$curr_backend" != 'mdb' ] && [ -n "$curr_backend" ]; then
```

Reply:
I think this can't happen: the "grep" would fail. and the script would exit. But I'm not sure.